modules/rostests/rosautotest/main.cpp

```diff
@@ -52,6 +53,9 @@ wmain(int argc, wchar_t* argv[])
 {
     int ReturnValue = 1;
 
+    wcscpy(TestName, argv[0]);
```

Comment:
`argv[0]` entirely depends on how the application was started, it could be `rosautotest.exe`, `c:\tests\rosautotest.exe` or even something like `..\rosautotest.exe`.
It would be more defined to use something like `GetModuleFileNameW(NULL, path, _countof(path))` and only use the filename of that

Reply:
Thanks, @learn-more. I did not realized this limitation of the argv[0] feature. I don't think that I have ever tried to use it programmatically before, but was aware that it returned the first part of the command line. The code has been modified to use the GetModuleFileNameW now.